docs/use.md

```diff
@@ -86,6 +86,7 @@ Notes:
         * `backup_start_time` (defaults to `"06:00"`)
         * `binlog` (defaults to `false` for 1st gen, true for 2nd gen, set to "true" to use)
         * `activation_policy` (defaults to `on demand`)
+        * `authorized_networks` (defaults to none)
```

Comment:
I'd clarify what format is expected here

Reply:
Good idea, thanks!